docker.local/bin/blobber.init.setup.sh

```diff
@@ -2,9 +2,9 @@
 
 for i in $(seq 1 6)
 do
-  mkdir -p docker.local/blobber$i
-  mkdir -p /mnt/ssd/blobber$i
-  mkdir -p /mnt/hdd/blobber$i
-  mkdir -p docker.local/validator$i
-  mkdir -p /mnt/hdd/validator$i
+  sudo mkdir -p docker.local/blobber$i
```

Comment:
this might not work if the process that runs this script doesn't have required permissions. For example in the case of editor. 

I would suggest an alternate approach: 
you could make a the script take additional command line parameters and when those commands those arguments are passed, the additional commands which require sudo permissions can run:
```
sudo mkdir -p /mnt/hdd/validator$i
sudo mkdir -p /mnt/ssd/blobber$i
sudo mkdir -p /mnt/hdd/blobber$i
```

Reply:
will address this in new thread